nest.py

```diff
@@ -1,5 +1,6 @@
 import requests
-from nest_secrets import PROJECT_ID, DEVICE_ID, CLIENT_ID, CLIENT_SECRET, REFRESH_TOKEN
```

Comment:
Leave this instead. We don't have a nest_secrets_private.py in the repo. nest_secrets.py is already in the repo similar to how Thu did it.

Reply:
haha, my bad. I changed on accident, I told Thu to do it like this and I forgot to do it myself.